backends/p4tools/testgen/lib/exceptions.h

```diff
@@ -0,0 +1,41 @@
+#ifndef TESTGEN_LIB_EXCEPTIONS_H_
+#define TESTGEN_LIB_EXCEPTIONS_H_
+
+#include "lib/exceptions.h"
+
+namespace P4Tools {
+
+namespace P4Testgen {
+
+/// This class indicates a feature that is not implemented in P4Testgen.
```

Comment:
I think this file should be called "unimplemented.h".

Reply:
There is a plan to add finer-grained exceptions. For example, exceptions that indicate that the tool has encountered a path that is not valid, exceptions for unimplemented features, or exceptions that are fatal. We are still using COMPILER_BUG, but those messages are confusing.